src/index.js

```diff
@@ -69,19 +69,21 @@ export default class StickyBox extends React.Component {
 
   initial() {
     const {bottom, offset} = this.props;
+    const offsetTop    = Array.isArray(offset) ? offset[0] : offset;
+    const offsetBottom = Array.isArray(offset) ? offset[1] : offset;
```

Comment:
Set the `else` case to `0` if you want to keep existing behaviour as much as possible.

Reply:
Can be shortened as `offset[1] || offset` but I liked being a bit verbose here.